index.html

```diff
@@ -25,15 +26,174 @@ <h1> I’m George
+                <li> <a href="https://github.com/george827"><img src="img/Vectorgit.png" alt="git"></a> </li>
+                <li> <a href="#"><img src="img/Vectorch.png" alt="icon"></a> </li>
+            </ul>
+        </div>
+    </section>
+
+    </section>
+
+    <section class="snapshot1">
+        <img src="img/Snapshoot-Portfolio1.png" alt="snapshot1" class="image-po">
+        <section id="main">
+
+            <div class="head-one">
+                <h1>Tonic</h1>
+            </div>
+                <div class="grid-container">
+                    <div class="grid-item"> <span class="canop">CANOPY</span></div>
+                    <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> Back End Dev</div>
+                    <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> 2015</div>  
+                </div>
+                
+            <div class="doc">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.                    
+                </p>
+            </div>
+            <div class="tags">
+                <ul class="works">
+                    <li> <a href="#" class="btn"> html </a></li>
+                    <li> <a href="#" class="btn"> css </a></li>
+                    <li> <a href="#" class="btn"> javascript </a></li>
+                </ul>
+            </div>
+
+                <a href="#"> <div class="btn-main">See Project</div> </a>             
+            
+        </section>
+    </section>
+
+    <section class="snapshot1">
+        <img src="img/Snapshoot-Portfolio2.png" alt="snapshot1" class="image-po">
+        <section id="main">
+
+            <div class="head-one-alt">
+                <h1>Multi-Post
+                    Stories</h1>
+            </div>
+                <section class="grid-container">
+                    <div class="grid-item"> <span class="canop">CANOPY</span></div>
+                    <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> Back End Dev</div>
+                    <div class="grid-item" id="grey-works"> <img src="img/Counter.png" alt="counter" class="counter"> 2015</div>  
+                </section>
+          
+            <div class="doc">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.                    
+                </p>
+            </div>
+            <div class="tags">
+                <ul class="works">
+                    <li> <a href="#" class="btn"> html </a></li>
+                    <li> <a href="#" class="btn"> css </a></li>
+                    <li> <a href="#" class="btn"> javascript </a></li>
+                </ul>
+            </div>
+
+            <a href="#"> <div class="btn-main">See Project</div> </a> 
+        </section>
+    </section>
```

Comment:
- [ ] Kindly ensure that yoou use the correct margin/ padding left and right to align the works cards. The margins from the page from the cards are thinner than the actual desired one. This then leads to a distortion  in the design. Kindly ensure that you update the project margins.

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/193088800-c99eb1e2-7194-46e4-90ab-85d426919d15.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/193088939-0e454fa9-d791-4b5e-bf46-37af5141ce0e.png)

**Kindly ensure that you make this change in all the works cards as well**